Tests/SourceKittenFrameworkTests/ModuleTests.swift

```diff
@@ -48,9 +48,31 @@ class ModuleTests: XCTestCase {
 #if SWIFT_PACKAGE
 extension ModuleTests {
     func testCommandantDocsSPM() {
-        let commandantPath = projectRoot + "/Packages/Commandant-0.11.3/"
+    #if swift(>=3.1) && os(Linux)
+        // FIXME
+        print("FIXME: Skip \(#function), because our sourcekitInProc on Swift 3.1 for Linux seems to be broken")
+    #else
+        func findCommandant(in directory: String) -> String? {
```

Comment:
It's a shame `swift build --get-package-path Commandant` (from [SE-0082](https://github.com/apple/swift-evolution/blob/master/proposals/0082-swiftpm-package-edit.md#detailed-design)) doesn't appear to work yet 🤔